Podcast/SupplierView.swift

```diff
@@ -9,9 +9,13 @@
 import UIKit
 import SnapKit
 
-class CardTableViewCellContextView: UIView {
+protocol SupplierViewDelegate: class {
+    func supplierViewDidPressFeedControlButton(supplierView: SupplierView)
+}
+
+class SupplierView: FeedElementSupplierView {
```

Comment:
Can we think of something to add in front of SupplierView, it's a little plain and could clash later if we make more supplier views.

Reply:
done.